packages/nc-gui/components/cell/Email.vue

```diff
@@ -70,8 +73,10 @@ watch(
     v-if="editEnabled"
     :ref="focus"
     v-model="vModel"
-    class="w-full outline-none text-sm px-1 py-2"
     :placeholder="isEditColumn ? '(Optional)' : ''"
+    class="w-full focus:!outline-none focus:ring-0 focus:border-none !border-none !outline-none text-sm px-2"
+    :type="isForm ? 'email' : 'text'"
+    :autocomplete="isForm ? 'email' : 'text'"
```

Comment:
define a variable and use it here. don't do repetitive stuff.

Reply:
small typo. fixed